cattle/utils.py

```diff
@@ -87,10 +87,17 @@ def get_containers(self):
         return self._get(self.url + '/containers')
 
     def get_latest_stat(self):
-        return self.get_containers()['stats'][-1]
+        containers = self.get_stats()
+        if containers:
```

Comment:
Looks like this is the only one comment that needs to be addressed. After its in, I will L..M :)